qiskit/optimization/algorithms/multistart_optimizer.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Defines an abstract class for multi start optimizers. A multi start optimizer is an optimizer
+that may run minimization algorithm for the several time with different initial guesses to achieve
+better results. This implementation is suitable for local optimizers."""
+
+import logging
+import time
+from abc import ABC
+from typing import Optional, Callable
+
+import numpy as np
+from scipy.stats import uniform
+
+from qiskit.optimization import QuadraticProgram, INFINITY
+from qiskit.optimization.algorithms import OptimizationAlgorithm, OptimizationResult
+
+logger = logging.getLogger(__name__)
+
+
+# we disable a warning: "Method 'a method' is abstract in class 'OptimizationAlgorithm' but
+# is not overridden (abstract-method) since this class is not intended for instantiation
+# pylint: disable=W0223
+class MultiStartOptimizer(OptimizationAlgorithm, ABC):
+    """
+    An abstract class that implements multi start optimization and should be used by
```

Comment:
Sentence seems unfinished. I could easily be ended before the 'and' I think and make enough sense

Reply:
Updated, something urgent disturbed me that time.